runners/katacoda/index.ts

```diff
@@ -21,9 +21,11 @@ export class Katacoda extends Runner {
     private setupScripts: KatacodaSetupScript[] = [];
     private assetManager: KatacodaAssetManager;
     private setupDir: string;
-    private currentDir: string = "/root";
+    private currentDir: string = path.join("/root");
+    private workingDir: string = path.join("/root");
     private terminalCounter: number = 1;
     private terminals: KatacodaTerminals[] = [{function: "default", terminalId: 1}];
+    private useDevonCommand: boolean = false; 
```

Comment:
We will use these variables in both runners. So we can move the variables to the runner base class.